code/modules/mob/living/basic/boss/boss.dm

```diff
@@ -0,0 +1,141 @@
+//not quite simple animal megafauna but close enough
+// port actual megafauna stuff once it gets used for lavaland megafauna
+//im using it for stuff both of them get
+/mob/living/basic/boss
+	combat_mode = TRUE
+	sentience_type = SENTIENCE_BOSS
+	mob_biotypes = MOB_ORGANIC|MOB_SPECIAL
+	faction = list(FACTION_MINING, FACTION_BOSS)
+	damage_coeff = list(BRUTE = 1, BURN = 1, TOX = 0, CLONE = 0, STAMINA = 0, OXY = 0)
+	obj_damage = 400
+	unsuitable_cold_damage = 0
+	unsuitable_heat_damage = 0
+	unsuitable_atmos_damage = 0
+	move_force = MOVE_FORCE_OVERPOWERING
+	move_resist = MOVE_FORCE_OVERPOWERING
+	pull_force = MOVE_FORCE_OVERPOWERING
+	mob_size = MOB_SIZE_HUGE
+	layer = LARGE_MOB_LAYER
+	mouse_opacity = MOUSE_OPACITY_OPAQUE
```

Comment:
I'll be honest, I know you're copying this line off the old megafauna path but I think this is best left done only for bosses which have a fairly small icon (e.g. the thing subtype). I recall having had some troubles clicking on other things around some of the bigger megafauna mobs when I had to.